### DISCUSSION TITLE
请问，想要针对Antd组件进行二次开发，请问应该如何做呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [ant-design/ant-design#44421] [Q&A]
请问，想要针对Antd组件进行二次开发，请问应该如何做呢？

现有一需求：
Tabs 结合 面包屑做一个组件。

但是我看antd上Tabs和面包屑都是分开的2个组件，请问如何进行二次开发生产一个带面包屑的Tabs组件呢？
https://ant-design.antgroup.com/components/tabs-cn

**Comment by arvinxx** [comment]
样式上可以基于 https://ant-design.github.io/antd-style 来搞
